limitador/tests/integration_tests.rs

```diff
@@ -537,7 +537,7 @@ mod test {
         }
 
         // We wait for the flushing period to pass so the counters are flushed in the cached storage
-        tokio::time::sleep(Duration::from_millis(3)).await;
+        tokio::time::sleep(Duration::from_millis(4)).await;
```

Comment:
I didn't fix this in my previous PR since I couldn't find an easy way to do so, we could tackle this later to avoid sleeping in tests